### DISCUSSION TITLE
Add to the documentation of Select2TagWidget that it accepts user input

### DISCUSSION THREAD

**Discussion by mardukbp** [codingjoe/django-select2#290] [Ideas]
The Select2TagWidget implements the oddly named [tagging](https://select2.org/tagging) feature of Select2, which allows for dynamic option creation.

This is a widget that some people [would like to use](https://stackoverflow.com/questions/60056395/how-to-extend-a-django-forms-choicefield-to-also-accept-text-and-search-for-clos) (myself included) , but since it is not properly documented they cannot find it and end up rolling their own.

**Comment by codingjoe** [comment]
Hi @mardukbp,

Thanks for reaching out and for your feedback. You know what they say. There are only two hard things left in computer science: cache invalidation & naming things.

I would be careful to rename the classes unless we add proper deprecation, but I am also not opposed.
What we definitely can do, is use a different language in the documentation, to ensure more search hits from users wanting to solve this use case.

Please don't hesitate to open a documentation PR. They are always most welcome!

Cheers,
Joe

**Comment by mardukbp** [comment]
I agree with you that renaming the classes is complicated. Improving the documentation is a better way. I just created the PR #291.

Thank you for your quick reply and your openness to contributions :)
